dockstore-webservice/src/test/java/io/swagger/api/impl/ToolsImplCommonTest.java

```diff
@@ -0,0 +1,285 @@
+package io.swagger.api.impl;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.common.collect.Table;
+import com.google.gson.Gson;
+import io.dockstore.common.Registry;
+import io.dockstore.common.SourceControl;
+import io.dockstore.webservice.DockstoreWebserviceConfiguration;
+import io.dockstore.webservice.core.SourceFile;
+import io.dockstore.webservice.core.Tag;
+import io.dockstore.webservice.core.ToolMode;
+import io.dockstore.webservice.core.Workflow;
+import io.dockstore.webservice.core.WorkflowMode;
+import io.dockstore.webservice.core.WorkflowVersion;
+import io.swagger.model.Tool;
+import io.swagger.model.ToolDescriptor;
+import io.swagger.model.ToolTests;
+import io.swagger.model.ToolVersion;
+import org.apache.commons.lang3.tuple.Pair;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author gluu
+ * @since 17/01/18
+ */
+public class ToolsImplCommonTest {
+    static final Gson gson = new Gson();
+    static final String PLACEHOLDER_CONTENT = "potato";
+    static DockstoreWebserviceConfiguration  actualConfig = new DockstoreWebserviceConfiguration();
+
+    @BeforeClass
+    public static void setup() {
+        actualConfig.setHostname("localhost");
+        actualConfig.setPort("8080");
+        actualConfig.setScheme("http");
+    }
+    static String json;
```

Comment:
FYI, not sure if I commented on the right line. 
But json here seems to only be used in one method. 
There's no point in making is static/global and it it may very well be leading to hidden behaviour
https://stackoverflow.com/questions/19158339/why-are-global-variables-evil

Reply:
Ok, I think I was looking at the other lines.  I'm moving the json back inside the test, but should I move the other lines back?

Reply:
I think the rule of thumb in the comment is valid. 
If its a static final constant then a global variable is fine. 
If its a global variable that is mutable and can be changed by many methods, then consider avoiding it, like in this case where it looks like it is not necessary.